planning/behavior_path_planner/include/behavior_path_planner/scene_module/scene_module_interface.hpp

```diff
@@ -95,6 +93,7 @@ class SceneModuleInterface
 
     const auto ns = std::string("~/debug/") + module_ns;
     pub_debug_marker_ = node.create_publisher<MarkerArray>(ns, 20);
+    pub_path_candidate_ = node.create_publisher<Path>("/planning/path_candidate/" + module_ns, 1);
```

Comment:
please move publishers to behavior_path_planner_node in order to make pub/sub interface clear.

Reply:
done
https://github.com/autowarefoundation/autoware.universe/pull/2486/commits/e06088d31948e76df716dd0289d8d6da372ba285